src/node_v8.cc

```diff
@@ -157,6 +157,20 @@ void CachedDataVersionTag(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(result);
 }
 
+void SetHeapSnapshotNearHeapLimit(const FunctionCallbackInfo<Value>& args) {
+  CHECK(args[0]->IsNumber());
```

Comment:
```suggestion
  CHECK(args[0]->IsUint32());
```

Reply:
But `options_->heap_snapshot_near_heap_limit` is `int64_t`.

Reply:
I think it would make sense to make it uint32_t, if necessary. It would be weird to set that value to over 2^31 anyway (typically a heapsnapshot is over 1MB - who needs 2PB of heapsnapshot, anyway?)

Reply:
Yes, maybe i also change `options_->heap_snapshot_near_heap_limit` to  `uint32_t`.